### PR TITLE
Mine blocks to a specific address

### PR DESCRIPTION
**Pull request by elraphty**
This feature allows a user to mine a certain amount of blocks to a specific address
@Evanfeenstra @kevkevinpal

### CODE REVIEW

File: src/modes/stack/app/src/AddNode.svelte [hunk 1]
```diff
@@ -42,10 +42,10 @@
...
     };
     return console.log(newNode);
-    stack.update((s) => ({
-      network: s.network,
-      nodes: [...s.nodes, newNode],
-    }));
+    // stack.update((s) => ({
+    //   network: s.network,
+    //   nodes: [...s.nodes, newNode],
+    // }));
```
Comment (kevkevinpal) [inline src/modes/stack/app/src/AddNode.svelte:48]: not sure why this is commented out on this PR

**Review by kevkevinpal — APPROVED** [review]
utACK Looking good to me just added a comment

File: src/modes/stack/app/src/AddNode.svelte (hunk 1 above)
Comment (elraphty) [reply src/modes/stack/app/src/AddNode.svelte]: Hello @kevkevinpal I have removed the commented code

**Comment by elraphty** [comment]
Good day @Evanfeenstra what are the requirements for the calculation of the Can Receive horizontal line bars?

**Pull request merged**